C++/search/Rabin_Karp.cpp

```diff
@@ -0,0 +1,59 @@
+#include<bits/stdc++.h>
+using namespace std;
+typedef long long int ll;
+//MODULAR EXPONENTIATION
+ll power(ll x,ll y,ll p)
+{
+	ll res=1;
+	x=x%p;
```

Comment:
Make use of Proper Variable Names, either Camel-Case/Snake-Case Naming Convention for better understanding of the Code.